scripts/linters/pylint_extensions.py

```diff
@@ -1893,6 +1893,204 @@ def visit_compare(self, node):
                 self.add_message('inequality-with-none', node=node)
 
 
+class DisallowedFunctionCallsChecker(checkers.BaseChecker):
+    """Custom pylint checker for language specific general purpose
+    regex checks of functions calls to be removed or replaced.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'disallowed-function-calls'
+    priority = -1
+    msgs = {
+        'C0031': (
+            'Please do not use %s.',
+            'remove-disallowed-function-calls',
+            'Please remove disallowed function calls.',
+        ),
+        'C0032': (
```

Comment:
Update the next id to use the value on top*

Reply:
Done